Makefile

```diff
@@ -49,8 +49,9 @@ build-cli-mac: ## Build the Mac CLI
 build-cli: build-cli-linux build-cli-mac ## Build the CLI
 
 init-package: ## Create the zarf init package, macos "brew install coreutils" first
-	$(ZARF_BIN) package create --confirm
-	mv zarf-init.tar.zst build
+	$(ZARF_BIN) package create --confirm --architecture amd64
```

Comment:
Do we want to try to be smart and only build the arch defined by the UNAME_S and UNAME_P?

Reply:
Not unless we are splitting init-package for dev/CI vs release.  Issue is we need to produce both on release.  I'm fine splitting that, but you actually don't need to specify arch if your building for your own arch, because zarf already figures that out internally